.pre-commit-config.yaml

```diff
@@ -25,3 +25,7 @@ repos:
           - --severity-level=medium
           - --confidence-level=medium
           - --skip=B608
+  - repo: https://github.com/syslabcom/slc.pre-commit
+    rev: main
```

Comment:
We need to pinpoint a version here because already yesterday there were failures because of problems with the package, see https://github.com/syslabcom/slc.pre-commit/pull/1